src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/Razor/ExtractToCodeBehindCodeActionResolver.cs

```diff
@@ -202,6 +202,11 @@ private static string GenerateCodeBehindClass(string className, string namespace
             .AddModifiers(CSharpSyntaxFactory.Token(CSharpSyntaxKind.PublicKeyword), CSharpSyntaxFactory.Token(CSharpSyntaxKind.PartialKeyword))
             .AddMembers(mock.Members.ToArray());
 
+        if (mock.CloseBraceToken.ContainsDirectives)
```

Comment:
Is it even worth having this condition? If there is a comment, or any other trivia, before the close brace, we would want to bring that across too.

Reply:
Great point! Comments wouldn't get brought across with this if condition here. I will remove the if.